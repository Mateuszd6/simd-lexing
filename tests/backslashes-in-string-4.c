struct foobar
{
    int a;
    int b;
    itn c;
    int d;
};

/* Move to the end */
int
main(int argc, char** argv)
{
    char const* foo = "And this invalid, because of three backslashes: \\\";

    long long long long long long long long long long long long
    long long long long long long long long long long long long
    long long long long long long long long long long long long
    long long long long long long long long long long long long;
};
