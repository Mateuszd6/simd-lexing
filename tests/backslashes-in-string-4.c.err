./tests/backslashes-in-string-4.c:13:77: error: Unescaped newline in string
