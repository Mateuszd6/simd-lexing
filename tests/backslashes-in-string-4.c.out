./tests/backslashes-in-string-4.c:1:1: TOK 0 (L = 6): "struct"
./tests/backslashes-in-string-4.c:1:8: TOK 0 (L = 6): "foobar"
./tests/backslashes-in-string-4.c:2:1: TOK 1 (L = 1): "{"
./tests/backslashes-in-string-4.c:3:5: TOK 0 (L = 3): "int"
./tests/backslashes-in-string-4.c:3:9: TOK 0 (L = 1): "a"
./tests/backslashes-in-string-4.c:3:10: TOK 1 (L = 1): ";"
./tests/backslashes-in-string-4.c:4:5: TOK 0 (L = 3): "int"
./tests/backslashes-in-string-4.c:4:9: TOK 0 (L = 1): "b"
./tests/backslashes-in-string-4.c:4:10: TOK 1 (L = 1): ";"
./tests/backslashes-in-string-4.c:5:5: TOK 0 (L = 3): "itn"
./tests/backslashes-in-string-4.c:5:9: TOK 0 (L = 1): "c"
./tests/backslashes-in-string-4.c:5:10: TOK 1 (L = 1): ";"
./tests/backslashes-in-string-4.c:6:5: TOK 0 (L = 3): "int"
./tests/backslashes-in-string-4.c:6:9: TOK 0 (L = 1): "d"
./tests/backslashes-in-string-4.c:6:10: TOK 1 (L = 1): ";"
./tests/backslashes-in-string-4.c:7:1: TOK 1 (L = 1): "}"
./tests/backslashes-in-string-4.c:7:2: TOK 1 (L = 1): ";"
./tests/backslashes-in-string-4.c:10:1: TOK 0 (L = 3): "int"
./tests/backslashes-in-string-4.c:11:1: TOK 0 (L = 4): "main"
./tests/backslashes-in-string-4.c:11:5: TOK 1 (L = 1): "("
./tests/backslashes-in-string-4.c:11:6: TOK 0 (L = 3): "int"
./tests/backslashes-in-string-4.c:11:10: TOK 0 (L = 4): "argc"
./tests/backslashes-in-string-4.c:11:14: TOK 1 (L = 1): ","
./tests/backslashes-in-string-4.c:11:16: TOK 0 (L = 4): "char"
./tests/backslashes-in-string-4.c:11:20: TOK 1 (L = 1): "*"
./tests/backslashes-in-string-4.c:11:21: TOK 1 (L = 1): "*"
./tests/backslashes-in-string-4.c:11:23: TOK 0 (L = 4): "argv"
./tests/backslashes-in-string-4.c:11:27: TOK 1 (L = 1): ")"
./tests/backslashes-in-string-4.c:12:1: TOK 1 (L = 1): "{"
./tests/backslashes-in-string-4.c:13:5: TOK 0 (L = 4): "char"
./tests/backslashes-in-string-4.c:13:10: TOK 0 (L = 5): "const"
./tests/backslashes-in-string-4.c:13:15: TOK 1 (L = 1): "*"
./tests/backslashes-in-string-4.c:13:17: TOK 0 (L = 3): "foo"
./tests/backslashes-in-string-4.c:13:21: TOK 1 (L = 1): "="
